hbase-client/src/main/java/org/apache/hadoop/hbase/client/TableDescriptorBuilder.java

```diff
@@ -884,7 +891,8 @@ public int getNormalizerTargetRegionCount() {
      */
     @Override
     public long getNormalizerTargetRegionSize() {
-      return getOrDefault(NORMALIZER_TARGET_REGION_SIZE_KEY, Long::valueOf, Long.valueOf(-1));
+      long target_region_size = getOrDefault(NORMALIZER_TARGET_REGION_SIZE_KEY, Long::valueOf, Long.valueOf(-1));
+      return target_region_size == Long.valueOf(-1) ? getOrDefault(NORMALIZER_TARGET_REGION_SIZE_MB_KEY, Long::valueOf, Long.valueOf(-1)) : target_region_size;
```

Comment:
Get the new configuration first? If both the new and old configurations exist, the new configuration value should be the correct value.